persistence.js

```diff
@@ -33,6 +31,8 @@ function RedisPersistence (opts) {
   this.maxSessionDelivery = opts.maxSessionDelivery || 1000
   this._db = new Redis(opts)
 
+  this.messageIdCache = {}
```

Comment:
pay attention that only unsuccessful delivery messageId's are leaking in cache key since clearMessageId is removing delivered packets from cache.
And his was only the draft idea. I wish to bring in tiny-lru or hashlru @mcollina had mentioned before